tests/system/action/projector/test_project.py

```diff
@@ -52,132 +53,166 @@ def test_project(self) -> None:
-            "projection/112",
-            {
-                "content_object_id": "meeting/1",
-                "current_projector_id": 65,
-                "meeting_id": 1,
-            },
+        self.assert_status_code(response, 200)
+
+    def test_temporary_user_as_content_object_okay(self) -> None:
+        """
+        Although the temporaray user has to have a group, this test checks the temporary user without group set
+        """
+        self.create_model("user/2", {"username": "temporary", "meeting_id": 1},)
+        response = self.request(
+            "projector.project", {"ids": [], "content_object_id": "user/2", "meeting_id": 1}
         )
+        self.assert_status_code(response, 200)
 
-    def test_meeting_as_content_object_error(self) -> None:
+    def test_project_without_meeting_id(self) -> None:
         response = self.request(
             "projector.project",
             {
-                "ids": [65],
-                "content_object_id": "meeting/2",
-                "stable": True,
-                "type": "current-list-of-speakers",
+                "ids": [23],
+                "content_object_id": "assignment/453",
             },
         )
         self.assert_status_code(response, 400)
+        self.assertIn("data must contain ['content_object_id', 'meeting_id', 'ids'] properties", response.json["message"])
+
+    def test_project_wrong_meeting_by_ids_and_object(self) -> None:
+        response = self.request(
+            "projector.project",
+            {"ids": [23], "content_object_id": "assignment/452", "meeting_id": 2, "stable": True},
+        )
+        self.assert_status_code(response, 400)
         self.assertIn(
-            "The following models do not belong to meeting 2: [FullQualifiedId('projector/65')]",
-            response.json["message"],
+            "The following models do not belong to meeting 2: [FullQualifiedId('assignment/452'), FullQualifiedId('projector/23')]",
+            response.json["message"]
+        )
+
+    def test_project_wrong_meeting_by_content_user(self) -> None:
+        self.create_model("user/2", {"username": "normal user", "group_$1_ids": [1], "group_$_ids": ["1"]})
+        response = self.request(
+            "projector.project",
+            {"ids": [], "content_object_id": "user/2", "meeting_id": 2, "stable": True},
+        )
+        self.assert_status_code(response, 400)
+        self.assertIn(
+            "The following models do not belong to meeting 2: [FullQualifiedId('user/2')]",
+            response.json["message"]
+        )
+
+    def test_project_wrong_meeting_by_content_meeting(self) -> None:
+        response = self.request(
+            "projector.project",
+            {"ids": [], "content_object_id": "meeting/1", "meeting_id": 2, "stable": True},
         )
+        self.assert_status_code(response, 400)
+        self.assertIn("The following models do not belong to meeting 2: [FullQualifiedId('meeting/1')]", response.json["message"])
```

Comment:
Is it possible to add a `__str__` or `__repr__` to `FullQualifiedId` so it just prints `meeting/1`?

Reply:
done